go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/alicebob/gopher-json v0.0.0-20180125190556-5a6b3ba71ee6 // indirect
 	github.com/alicebob/miniredis v0.0.0-20180911162847-3657542c8629
 	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da // indirect
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
```

Comment:
why this dependency was added?

Reply:
I am looking into it....no idea how it got in

Reply:
This is still not removed.

Reply:
I copied from master... not sure why is it getting added...fixing it right away

Reply:
I copied my go.mod and go.sum from the master and it shows as different even though the files are same...